components/ResultsPage/YourAnswers.tsx

```diff
@@ -37,9 +37,7 @@ export const YourAnswers: React.VFC<{
                   href={`eligibility#${input.key}`}
                   text={tsln.resultsPage.edit}
                   target="_self"
-                  ariaLabel={`${tsln.resultsPage.edit} ${
-                    tsln.resultsQuestions[input.key]
-                  }`}
+                  ariaLabel={tsln.resultsPage.yourAnswers[input.key]}
```

Comment:
I think what Jeremy wanted us to use is
` tsln.resultsQuestions[input.key] ` 
I think you should add the new text to the resultsQuestions object instead

Reply:
In en.ts, we can find "resultsQuestions: **apiEn.questionShortText**", therefore if we want to still use resultsQuestions, we have to add new translations into **questionShortText**. My concern is that is it good to do so? Because what we are going to add are not questions nor short version of questions, we are just adding aria labels.